staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -803,6 +803,23 @@ type KubeletConfiguration struct {
 	// Default: true
 	// +optional
 	EnableSystemLogHandler *bool `json:"enableSystemLogHandler,omitempty"`
+	// lockFilePath is the path that kubelet will use to as a lock file.
+	// It uses this file as a lock to synchronize with other kubelet processes
+	// that may be running.
+	// Dynamic Kubelet Config (beta): If dynamically updating this field, consider that
+	// it may impact the ability of the Kubelet to interact with other Kubelet instances.
+	// Default: ""
+	// +optional
+	LockFilePath string `json:"lockFilePath,omitempty"`
+	// exitOnLockContention is a flag that signifies to the kubelet that it is running
+	// in "bootstrap" mode. This requires that 'LockFilePath' has been set.
```

Comment:
> This requires that 'LockFilePath' has been set.

is this validated in API validation (which is what helps determine if a new kubelet config file is valid or not) or at runtime?

Reply:
It's being validated in runtime, I'm moving it to the API validation.